scripts/pre_commit_linter.py

```diff
@@ -0,0 +1,151 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Pre-commit script for Oppia.
+
+This script uses the JSCS node module to lint JavaScript code
+It returns and autofix errors
```

Comment:
It prints a list of lint errors to the terminal. If the --autofix flag is passed, it can also autofix these errors.